CondTools/Ecal/python/copySimComponentShapeFromFiles_cfg.py

```diff
@@ -0,0 +1,63 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("TEST")
+process.load("CalibCalorimetry.EcalTrivialCondModules.EcalTrivialCondRetriever_cfi")
+process.EcalTrivialConditionRetriever.producedEcalSimComponentShape = cms.untracked.bool(True)
+process.EcalTrivialConditionRetriever.getSimComponentShapeFromFile = cms.untracked.bool(True) ### if set False hadrdcoded shapes will be loaded by default
+
+
+### phase II Pulse Shapes
+process.EcalTrivialConditionRetriever.sim_component_shape_TI = cms.untracked.double(1)
+process.EcalTrivialConditionRetriever.sim_component_shape_EB_thresh = cms.double(0.00013)
+fileNames = [f"EB_SimComponentShape_PhaseI_depth{i}.txt" for i in range(0,23)]
+#fileNames = [f"EB_SimComponentShape_PhaseII_depth{i}.txt" for i in range(0,23)]
+process.EcalTrivialConditionRetriever.EBSimComponentShapeFiles = cms.untracked.vstring(fileNames)
+
+
+process.load("CondCore.CondDB.CondDB_cfi")
+#process.CondDBCommon.connect = 'oracle://cms_orcon_prod/CMS_COND_31X_ECAL'
+#process.CondDBCommon.DBParameters.authenticationPath = '/nfshome0/popcondev/conddb'
+process.CondDB.connect = 'sqlite_file:EBSimComponentShape_PhaseI.db'
```

Comment:
and then when the upload to the DB is done, this config should read it from the GT not a local sqlite file

Reply:
The db is uploaded. It can be found here:
https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/tags/EcalSimComponentShape_PhaseII

Reply:
The db is uploaded. It can be found here:
https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/tags/EcalSimComponentShape_PhaseII

Reply:
@wang0jin thanks, we need two more things: 
 (1) the Phase-1 tag
 (2) an announcement to https://cms-talk.web.cern.ch/c/ppd/alca/108 in which requet this to be put into the 131X queues, well actually the queue depends if this will be backported. @jhakala will it be backported? if yes, to which release?

Reply:
> will it be backported? 

well actually this might be too big of a PR to be backported... the question is which MC production will use this development.

Reply:
Hi @tvami, I don't know the answer to that exactly -- I think Thomas was discussing this with possible "consumers" of samples made with these new digis. @thomreis could you give your input on what would be useful to any interested parties?

From my side, I would be OK with not backporting this and waiting for some future Phase-II campaign.

Reply:
Hi, it was @AdrianoDee who was intersted to eventually have this in an upcoming production for the HLT upgrade. I do not know which release is foreseen for that though. Perhaps there is no backport needed.

Reply:
Since the upload to the db is now done, could this then move off the sqlite file?

Reply:
ahh I misread this file, I thought it read in the DB file, as opposed to being the one that writes it. In this case all this is fine, sorry for the noise